src/main/drivers/pwm_output.h

```diff
@@ -30,9 +30,9 @@ typedef enum {
     PWM_TYPE_DSHOT150,
```

Comment:
This will still break configurator, since the order in the original code was 600, 300, 150.